### PR TITLE
➕ add TypeScript

### PR DESCRIPTION
**Pull request by mewiemers**
https://nextjs.org/docs/basic-features/typescript

-  rename .js to .tsx and .ts
- add tsconfig.json
- added  Typescript devdependencies

### CODE REVIEW

**Review by m-wiemers — APPROVED** [review]
yeah, ride on 🐎

**Review by saschahaselhuhn — APPROVED** [review]
gj

**Review by SilviaBarcellini — APPROVED** [review]
lgtm 👍

File: tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "jsx": "preserve"
+  },
+  "include": [
+    "next-env.d.ts",
+    "**/*.ts",
+    "**/*.tsx"
+  ],
+  "exclude": [
+    "node_modules"
+  ]
```
Comment (phlgr) [inline tsconfig.json:28]: Is this prettified?

**Review by phlgr — CHANGES_REQUESTED** [review]

File: tsconfig.json (hunk 1 above)
Comment (mewiemers) [reply tsconfig.json]: now is prettified

**Pull request merged**